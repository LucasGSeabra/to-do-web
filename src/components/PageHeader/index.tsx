import React from 'react';

import './styles.css';

interface PageHeaderProps {
  name: string;
  small: string;
}

function PageHeader({ name, small }: PageHeaderProps) {
  return (
    <header className="page-header">
      <h2>
        {name}
        {' '}
        <small>{small}</small>
      </h2>
    </header>
  );
}

export default PageHeader;
