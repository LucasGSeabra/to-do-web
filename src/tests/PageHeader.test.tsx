/* eslint-disable no-undef */
import React from 'react';
import renderer from 'react-test-renderer';
import PageHeader from '../components/PageHeader';

describe('PageHeader component', () => {
  it('should render PageHeader correctly', () => {
    const component = renderer.create(<PageHeader name="Page" small="Header" />).toJSON();
    expect(component).toMatchSnapshot();
  });
});
