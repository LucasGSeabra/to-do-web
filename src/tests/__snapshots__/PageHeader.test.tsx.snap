// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageHeader component should render PageHeader correctly 1`] = `
<header
  className="page-header"
>
  <h2>
    Page
     
    <small>
      Header
    </small>
  </h2>
</header>
`;
